logger/logger.go

```diff
@@ -0,0 +1,27 @@
+package logger
+
+import (
+	"os"
+
+	runtime "github.com/banzaicloud/logrus-runtime-formatter"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// SetLogger configures the logrus logger and sets the log level
+func SetLogger() {
+	if os.Getenv("DEBUG") == "true" {
+		log.SetLevel(log.DebugLevel)
+	} else {
+		log.SetLevel(log.InfoLevel)
+	}
+
+	log.SetFormatter(&runtime.Formatter{
```

Comment:
Ok so feel free to ask me to revert this. I added a plugin to the logger that enabled reporting the file. line number and function that a log event occurred on. It made it easier to troubleshoot issues.

```
time="2021-02-09 23:18:21" level=info msg="Added Slack client: xoxb-63923..." file=/Users/ryancurrah/git/chatto/channels/slack/slack.go function=NewChannel line=50
time="2021-02-09 23:18:21" level=info msg="Registered CacheStoreFSM" file=/Users/ryancurrah/git/chatto/fsm/store.go function=LoadStore line=161
time="2021-02-09 23:18:21" level=info msg="* TTL:    -1" file=/Users/ryancurrah/git/chatto/fsm/store.go function=LoadStore line=162
time="2021-02-09 23:18:21" level=info msg="* Purge:  -1" file=/Users/ryancurrah/git/chatto/fsm/store.go function=LoadStore line=163
time="2021-02-09 23:18:21" level=info msg="Server started" file=/Users/ryancurrah/git/chatto/bot/server.go function=ServeBot line=142
time="2021-02-09 23:18:21" level=info msg="Connecting to Slack with Socket Mode..." file=/Users/ryancurrah/git/chatto/channels/slack/slack.go function=func1 line=168
time="2021-02-09 23:18:23" level=info msg="Connected to Slack with Socket Mode" file=/Users/ryancurrah/git/chatto/channels/slack/slack.go function=func1 line=172
```

Reply:
How about enabling the `runtime.Formatter` only when `DEBUG` is set to `true`?
That way, in a non-debugging environment, the logs are cleaner.
Great addition, by the way.

Reply:
Good call. I will do that.